composer.json

```diff
@@ -21,7 +21,9 @@
         "symfony/filesystem": "^6.3",
         "symfony/framework-bundle": "^6.3",
         "symfony/phpunit-bridge": "^6.3",
-        "phpstan/phpstan": "1.11.x-dev"
+        "phpstan/phpstan": "1.11.x-dev",
+        "phpstan/phpstan-symfony": "1.4.x-dev",
+        "friendsofphp/php-cs-fixer": "^3.38"
```

Comment:
What is the purpose of this? I don't think it's needed, because we don't use it as a dependency and install it via third-party on CI. Locally, it's probably a good idea to have it installed globally, not per project.

In other words, we don't "depend" on those libs directly, they are just used on CI to help, but nothing more. It seems best practice to not require it in composer.json anymore. Actually, I think the same relates to PHPStan as well, we can completely remove those 2 libs and if CI is happy - double win, fewer dependencies.

Reply:
The idea was to prevent checks to fail before a PR was created and since `phpstan` was already a dev-dependency, but changed it.

I've now added only `phpstan/phpstan-symfony` as requested.